simapp/test_helpers.go

```diff
@@ -231,21 +238,11 @@ func createIncrementalAccounts(accNum int) []sdk.AccAddress {
 func AddTestAddrsFromPubKeys(app *SimApp, ctx sdk.Context, pubKeys []cryptotypes.PubKey, accAmt sdk.Int) {
 	initCoins := sdk.NewCoins(sdk.NewCoin(app.StakingKeeper.BondDenom(ctx), accAmt))
 
-	setTotalSupply(app, ctx, accAmt, len(pubKeys))
```

Comment:
why we don't set total supply any more?

Reply:
setTotalSupply relied on SetSupply method which is part of the bank unsafe api. it was removed for this reason. based on how simapp is structured supply should be set via genesis only.